src/stactools/sentinel2/constants.py

```diff
@@ -103,52 +112,75 @@
 # available for each band as separate assets.
 # The first resolution is the sensor gsd; others
 # are downscaled versions.
-BANDS_TO_RESOLUTIONS = {
-    'B01': [
-        60,
-    ],
-    'B02': [
-        10,
-        20,
-        60,
-    ],
-    'B03': [
-        10,
-        20,
-        60,
-    ],
-    'B04': [
-        10,
-        20,
-        60,
-    ],
-    'B05': [
-        20,
-        60,
-    ],
-    'B06': [
-        20,
-        60,
-    ],
-    'B07': [
-        20,
-        60,
-    ],
-    'B08': [
-        10,
-    ],
-    'B8A': [
-        20,
-        60,
-    ],
+BANDS_TO_RESOLUTIONS: Final[Dict[str, List[int]]] = {
+    'B01': [60],
+    'B02': [10, 20, 60],
+    'B03': [10, 20, 60],
+    'B04': [10, 20, 60],
+    'B05': [20, 60],
+    'B06': [20, 60],
+    'B07': [20, 60],
+    'B08': [10, 20, 60],
+    'B8A': [20, 60],
     'B09': [60],
     'B10': [60],
-    'B11': [
-        20,
-        60,
-    ],
-    'B12': [
-        20,
-        60,
-    ],
+    'B11': [20, 60],
+    'B12': [20, 60],
 }
+
+L2A_IMAGE_PATHS: Final[List[str]] = [
```

Comment:
for the sinergise dataset, there's no metadata about what assets are available. We assume they all exist. Alternative is to have to do a list operation against S3 or to pass in all of the known assets from the S3 Inventory and only do the S3 query if all of the expected ones are not there.